WordPressCom-Stats-iOS/UI/NSObject+StatsBundleHelper.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface NSObject (StatsBundleHelper)
+
+@property (nonatomic, readonly) NSBundle *statsBundle;
```

Comment:
I was thinking about UIKit level but then realized bundles live in Foundation. A class extension may also not be the best approach... Maybe a bundle helper class...